src/tasks/minions/monsterActivity.ts

```diff
@@ -199,30 +200,39 @@ export const monsterTask: MinionTask = {
 		const isOnTaskResult = await isOnSlayerTask({ user, monsterID, quantityKilled: quantity });
 
 		const superiorTable = isOnTaskResult.hasSuperiorsUnlocked && monster.superior ? monster.superior : undefined;
-		const isInCatacombs = !usingCannon ? monster.existsInCatacombs ?? undefined : undefined;
+		const isInCatacombs = (!usingCannon ? monster.existsInCatacombs ?? undefined : undefined) && !isInWilderness;
+
+		const ringOfWealthI = (user.gear.wildy.hasEquipped('Ring of wealth (i)') && isInWilderness) as boolean;
```

Comment:
prefer starting boolean variable names with is/has/was/etc. Seeing this variable later, we will probably think its the ROW item or something else.